task-management/src/main/webapp/skin/css/tasks.less

```diff
@@ -1519,7 +1519,7 @@
       //border-right: 5px solid @successColor ~'!important; /** orientation=rt */ ';
     }
     .taskNonePriority {
-      border-left: 5px solid @primaryColor ~'!important; /** orientation=lt */ ';
+      border-left: 5px solid #476A9C ~'!important; /** orientation=lt */ ';
```

Comment:
literal color definition is forbidden, please consider using one of the variables defined here https://github.com/Meeds-io/platform-ui/blob/develop/platform-ui-skin/src/main/webapp/skin/less/variables.less

BTW, you will have to ask the Designer to pick a meaningful variable, if it's not the primary color, is it the secondary, tertiary of info color.

BTW, the eXo Branding colors are defined here:
https://github.com/exoplatform/platform-private-distributions/blob/develop/plf-enterprise-welcome-screens-webapp/src/main/webapp/WEB-INF/conf/welcome-screen/branding-configuration.xml#L56-L58